src/components/charts/chart-options.js

```diff
@@ -37,7 +37,7 @@ class ChartOptions {
                     fontWeight: 'normal',
                 },
             },
-            // chart title is rendered in wagtail
+            // Remove chart title
```

Comment:
Suggestion: `// Remove the chart title as rendered by Highcharts, as this is rendered in the surrounding component`